targets/TARGET_STM/TARGET_STM32WB/TARGET_STM32WB5MxG/TARGET_DISCO_WB5MMG/PinNames.h

```diff
@@ -27,6 +27,13 @@
 extern "C" {
 #endif
 
+void RGB_LED_OFF(void);
```

Comment:
This should follow the function names and be all lower case `void rgb_led_off(void)` as it's HAL target extension?

Reply:
@0xc0170 Done